parser.y

```diff
@@ -8965,6 +8966,32 @@ BlobType:
 		x := types.NewFieldType(mysql.TypeMediumBlob)
 		$$ = x
 	}
+|	"TINYTEXT" "BYTE"
```

Comment:
add a rule : `ByteOpt`？

Reply:
These rules should be added to `TextType`.